spec/support/auth_helpers.rb

```diff
@@ -8,7 +8,7 @@ def sign_in(space, person)
 
     authentication_method.bump_one_time_password!
 
-    post(space_authenticated_session_path(space),
```

Comment:
was `space_authenticated_session_path` deleted with the deletion of client?

Reply:
No, but it needs to use the `polymorphic_path` so that it works when on the `NeighborhoodsController#show` action.